include/sea_dsa/support/RemovePtrToIntStores.hh

```diff
@@ -0,0 +1,28 @@
+#ifndef SEA_DSA_REMOVE_PTR_TO_INT_STORES
+#define SEA_DSA_REMOVE_PTR_TO_INT_STORES
+
+#include "llvm/Pass.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+
+namespace sea_dsa {
+
+struct RemovePtrToIntStores : public llvm::ModulePass {
+  static char ID;
+
+  RemovePtrToIntStores() : llvm::ModulePass(ID) {}
+
+  bool runOnModule(llvm::Module &M);
+  bool runOnFunction(llvm::Function &F);
+  void getAnalysisUsage(llvm::AnalysisUsage &AU) const;
+
+  virtual const char *getPassName() const {
+    return "sea-dsa: remove ptrtoint stores";
+  }
+};
+
+llvm::ModulePass *CreateRemovePtrToIntStoresPass();
```

Comment:
This method is usually defined in another header file so that implementation details of the pass are not exposed to the clients.

Reply:
... and it's unused anyway now...

Reply:
then remove it from a header file. We will add it when it is actually used. Keep the method in cpp file though.